apps/stripe/src/app/api/saleor/transaction-initialize-session/use-case.ts

```diff
@@ -1,13 +1,20 @@
 import { buildSyncWebhookResponsePayload } from "@saleor/app-sdk/handlers/shared";
 import { err, ok } from "neverthrow";
+import Stripe from "stripe";
 
 import { TransactionInitializeSessionEventFragment } from "@/generated/graphql";
 import { BaseError } from "@/lib/errors";
 import { createLogger } from "@/lib/logger";
 import { AppConfigRepo } from "@/modules/app-config/app-config-repo";
 import { SaleorApiUrl } from "@/modules/saleor/saleor-api-url";
+import { SaleorMoney } from "@/modules/saleor/saleor-money";
+import { StripeMoney } from "@/modules/stripe/stripe-money";
 import { IStripePaymentIntentsApiFactory } from "@/modules/stripe/types";
 
+type TransactionInitializeSessionResponseData = {
+  stripeClientSecret: string | null;
+};
```

Comment:
this will be public contract, let's ensure we eventually document this and this makes sense for us

(null, or missing field, or empty data?)

Reply:
Done in https://github.com/saleor/apps/pull/1851/commits/70500c9777ef174b2814e501adb64d90e11be615 - now app will throw if client secret is null - it means that there is problem with stripe configuration